flink-runtime-web/web-dashboard/bower.json

```diff
@@ -21,7 +20,9 @@
     "dagre-d3": "0.4.17",
     "font-awesome": "4.5.0",
     "moment-duration-format": "1.3.0",
-    "qtip2": "2.2.1"
+    "qtip2": "2.2.1",
+    "angular-drag-and-drop-lists": "^1.4.0",
+    "nvd3": "^1.8.4"
```

Comment:
qtip2, angular-drag-and-drop-lists is MIT license
nvd3 is Apache License v2
These licenses are compatible to ours and we can use them.